src/main/java/org/roy/loadx/transaction/TransactionPrinter.java

```diff
@@ -1,59 +1,101 @@
 package org.roy.loadx.transaction;
 
 import java.util.ArrayList;
-import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
 
 public class TransactionPrinter {
-	private final TransactionAggregator transactionAggregator;
+	static private class TransactionDataSummaryEntry {
+		private String transactionName;
+		private String average;
+		private String count;
+		
+		public TransactionDataSummaryEntry(String transactionName, String average, String count) {
+			this.transactionName = transactionName;
+			this.average = average;
+			this.count = count;
+		}
 
-	public TransactionPrinter(TransactionAggregator transactionAggregator) {
-		this.transactionAggregator = transactionAggregator;
-	}
+		public String getTransactionName() {
+			return transactionName;
+		}
 
-	private static void println() {
-		System.out.println();
-		System.out.flush();
+		public String getAverage() {
+			return average;
+		}
+
+		public String getCount() {
+			return count;
+		}
 	}
+	
+	static private class TransactionDataSummary {
+		static private final String TRANSACTION_HEADER = "Transaction";
+		static private final String AVERAGE_HEADER = "Average";
+		static private final String COUNT_HEADER = "Count";
+		static private final int MAX_ROW_WIDTH = 78;
+		static private final String ROW_FORMAT = "%%-%ds | %%10s | %%10s%n";
+		static private final int PRE_ALLOCATED_SPACE_PER_ROW = 26;
+		
+		private int maxTransactionNameLength;
```

Comment:
Don't think this can be final since the value is possibly modified every time a `TransactionDataSummaryEntry` is added.  The value is updated as new entries are added to the `TransactionDataSummary` instance (lines 51-53).  And since the value is computed based on the length of the transaction names in each instance, I don't think this can be a static member either.

Perhaps renaming the variable to `largestTransactionNameLength` would be a better choice, but don't think this can be a static final.